usecases/objects/batch_references_add.go

```diff
@@ -44,8 +44,6 @@ func (b *BatchManager) AddReferences(ctx context.Context, principal *models.Prin
 	}
 	defer unlock()
 
-	ctx = classcache.ContextWithClassCache(ctx)
```

Comment:
**question (if-minor):** is this intended to be removed ? given that it's not removed from other endpoints

Reply:
Yes just above line 39